plugins/modules/packaging/os/apk.py

```diff
@@ -24,7 +24,7 @@
       - During upgrade, reset versioned world dependencies and change logic to prefer replacing or downgrading packages (instead of holding them)
         if the currently installed package is no longer available from any repository.
     type: bool
-    default: 'no'
+    default: False
```

Comment:
Please use `no` (without quotes) or `false` for YAML booleans. (And `yes` / `true` for true values.)

Probably just keeping `no` (without quotes) and `yes` is best, since that seemed to be the common style in these modules.